app/frontend/src/components/Pill/Pill.tsx

```diff
@@ -18,7 +19,7 @@ export interface PillProps {
   variant?: "rounded" | "square";
 }
 
-export default function Pill({ children, variant = "rounded" }: PillProps) {
+export default function Pill({ children, variant = "square" }: PillProps) {
```

Comment:
```suggestion
export default function Pill({ children, variant = "rounded" }: PillProps) {
```
That's what I meant, so you don't need to change the function component signature at all.

Reply:
Ok cool, I made the change.